### PR TITLE
fix(select): fix panel close animation

### PR DESCRIPTION
**Pull request by kara**
Tweaks to make the menu close animation smoother.

r: @hansl

### CODE REVIEW

File: src/lib/select/select-animations.ts [hunk 1]
```diff
@@ -58,10 +58,10 @@ export const transformPanel: AnimationEntryMetadata = trigger('transformPanel',
       width: '100%',
       transform: `translate3d(0, 0, 0) scaleY(0)`
     }),
-    animate(`150ms cubic-bezier(0.55, 0, 0.55, 0.2)`)
+    animate(`150ms cubic-bezier(0.25, 0.8, 0.25, 1)`)
```
Comment (hansl) [inline src/lib/select/select-animations.ts]: Where did you get those values?  Seems like a more aggressive ease-out.

File: src/lib/select/select.ts [hunk 2]
```diff
@@ -158,7 +159,6 @@ export class MdSelect implements AfterContentInit, OnDestroy {
       this._focusCorrectOption();
       this.onOpen.emit();
     } else {
-      this._focusHost();
```
Comment (hansl) [inline src/lib/select/select.ts]: Please change the comment above to reflect the code.

File: src/lib/select/select.ts [hunk 3]
```diff
@@ -108,6 +108,7 @@ export class MdSelect implements AfterContentInit, OnDestroy {
   /** Closes the overlay panel. */
   close(): void {
     this._panelOpen = false;
+    this._focusHost();
```
Comment (hansl) [inline src/lib/select/select.ts]: Please change the comment above to reflect the code.

File: src/lib/select/select.scss [hunk 4]
```diff
@@ -16,6 +16,7 @@ md-select {
   height: $md-select-trigger-height;
   min-width: $md-select-trigger-min-width;
   cursor: pointer;
+  user-select: none;
```
Comment (hansl) [inline src/lib/select/select.scss]: ?

File: src/lib/select/select-animations.ts (hunk 1 above)
Comment (kara) [inline src/lib/select/select-animations.ts]: These are the Material 1 values for the ease out animation.

**Pull request merged**